src/service-manager.js

```diff
@@ -124,7 +124,6 @@ class ServiceManager {
     // this overwrites values set in the config file
     const customEnv = {
       API_CONFIG_FILE: config.apiConfigFile || '',
-      CONNECTOR_ROUTE_BROADCAST_INTERVAL: 10000,
```

Comment:
I put it in env.list (e.g. `https://github.com/interledgerjs/five-bells-integration-test/blob/da-ilpkit-integration-tests/src/tests/data/kit1-env.list`). Makes it more transparent and future test cases can easier define a custom route broadcast interval.